### DISCUSSION TITLE
CCL is set to 50A - how to change?

### DISCUSSION THREAD

**Discussion by R0Li84** [Louisvdw/dbus-serialbattery#300] [Q&A] [answered]
I'm using SerialBattery(Jkbms) v0.13 and try to increase the charging current (CCL) which is set (under Parameters) to 50A. This parameter is only adjusting when the SOC is raising or lowering, the maximum is always 50A.

I have set the charging current in JKBMS to 120A and also in DVCC it is set to 120A. However, if I switch off DVCC and turn it back on, it always shows 50A. 

Can you please let me know where you take the 50A from and how I can change it? During short winter days, I can only charge my battery within a few hours per day. I would like to use the full power here, in case that the sun shines. With smart shunt, up to 100A was never an issue.

**Comment by Louisvdw** [comment, marked as answer]
Hi
See https://github.com/Louisvdw/dbus-serialbattery/wiki/How-to-install#how-to-increase-the-default-limits

**Reply by R0Li84** [reply]
Thanks! Wouldn't it be possible to change this parameter e.g. when you adjust charging current in DVCC?

**Reply by Louisvdw** [reply]
For now the only way to change it is to edit the utils.py value, but I am busy with a GUI so that you can set it up there.
See a preview https://ko-fi.com/post/Feature-update-20221127-K3K8GNNBB
